.circleci/config.yml

```diff
@@ -245,6 +245,7 @@ jobs:
             yarn run lint
             yarn run typecheck
             bundle exec rubocop --parallel
+            bundle exec brakeman
```

Comment:
(maybe for a future PR instead of this one) WDYT of some sort of `make all_lints` command or something so we can try to keep these in sync better?

Reply:
> (maybe for a future PR instead of this one) WDYT of some sort of `make all_lints` command or something so we can try to keep these in sync better?

Looks like it may exist already? Not sure if we have any reason not to just put all this in there.

https://github.com/18F/identity-idp/blob/66d9f9edce16a564d0d12849215a8b577ff3b145/Makefile#L24-L32

Reply:
ah that's a great catch, I will move all of these commands there

Reply:
Moved in 9ebdfc0